db/memtable.h

```diff
@@ -387,6 +389,12 @@ class MemTable {
 
   Env* env_;
 
+  // Extract sequential insert prefixes.
+  const SliceTransform* insert_prefix_extractor_;
+
+  // Insert hints for each prefix.
+  std::unordered_map<Slice, MemTableInsertHint, SliceHasher> insert_hints_;
```

Comment:
I'm using the same Arena object passing to InlineSkipList to allocate the hitns. My understanding is the memory usage will be counted in this way, but I'll double check.